daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2471,6 +2472,7 @@ func (devices *DeviceSet) Status() *Status {
 		}
 	}
 
+	status.MinFreeSpace = fmt.Sprintf("%v%%", devices.minFreeSpacePercent)
```

Comment:
@thaJeztah So existing format is fine? I am not sure what's the suggestion.

Or you are suggesting that we create two entries. One for percentage and one for absolute value.

Thin Pool Minimum Free Space Percent: X
Thin Pool Minimum Free Space: Y MB